app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -340,11 +340,11 @@ class MainActivity : AppCompatActivity(),
 
         // initialize the text & HTML
         if (!isRunningTest) {
-            aztec.sourceEditor.displayStyledAndFormattedHtml(EXAMPLE)
+            aztec.sourceEditor?.displayStyledAndFormattedHtml(EXAMPLE)
         }
 
         if (savedInstanceState == null) {
-            aztec.visualEditor.fromHtml(aztec.sourceEditor.getPureHtml())
+            aztec.visualEditor.fromHtml(aztec.sourceEditor?.getPureHtml()!!)
```

Comment:
No problem ;). 

You'd have to add a check so that `fromHtml()` method is not called with a `null` value (or change it so that it can handle it). In this case it wouldn't we'd need to initialize the visual editor with `EXAMPLE`.